src/Controller/Admin/BoardCrudController.php

```diff
@@ -58,8 +69,14 @@ public function configureFields(string $pageName): iterable
         yield IntegerField::new('hearingResponseDeadline', 'Hearing response deadline(days)');
         yield IntegerField::new('finishHearingDeadlineDefault', 'Finish hearing deadline(days)');
         yield IntegerField::new('finishProcessingDeadlineDefault', 'Finish processing case deadline(days)');
-        yield TextareaField::new('complainantPartyTypes', 'Complainant party types');
-        yield TextareaField::new('counterPartyTypes', 'Counter party types');
-        yield TextareaField::new('statuses', 'Statuses');
+        yield TextareaField::new('complainantPartyTypes', 'Complainant party types')
+            ->setHelp($this->translator->trans('Configuration of complainant types. Each line will be treated as a type of which the first will be the one used for sorting purposes.', [], 'admin'))
```

Comment:
I find the sentence “Configuration of complainant types. Each line will be treated as a type of which the first will be the one used for sorting purposes.” a little hard to understand.

Consider rewording to “List of complainant party types. One per line. The first party type can be used for sorting cases.”, say.

Reply:
Reworded helper.